### ISSUE TITLE
bug: selected channel search result is not added to channel list upon selection

### ISSUE THREAD

**Issue by MartinCupela** [GetStream/stream-chat-react#1684]
**Describe the bug**

Only direct-messages channels created upon clicking the search result are added into the channel list.

**To Reproduce**

1. Setup channel search filters
```
<ChannelList filters={filters} sort={sort} options={options} showChannelSearch additionalChannelSearchProps={
      {searchForChannels: true,
        searchQueryParams: {
        channelFilters: {
          filters: { members: { $in: [userId ] } },
        }}}}
    />
```
2. Type string contained in a name of at least one channel
3. Select a channel from the search result list
4. Observe that the channel content is rendered in the message list but the channel preview is not visible in the channel list.

**Expected behavior**

Add selected channel from the search list into the channel list. The position should be configurable - alphabetical order / top of the list.

**Video**

[searched_channel_not_added_to_channel_list.webm](https://user-images.githubusercontent.com/32706194/180363903-115be293-6ae3-48b6-be71-79ed0cdcc5de.webm)

**Package version**
- stream-chat-react: v9.1.5

**Comment by MartinCupela** [comment]
this will be fixed with the introduction of theming v2 changes

**Comment by MartinCupela** [comment]
search result addition to channel list fixed with release of [v10.0.0](https://github.com/GetStream/stream-chat-react/releases/tag/v10.0.0)
